yucca/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: yucca_yucca_1
+      APP_PORT: 9910
+
+  yucca:
+    image: yuccastream/yucca:0.10.0@sha256:9551f42cf20aa1d11bf0df6ba5996cc10f506c8225a60a448876cebefe539eb2
+    restart: on-failure
+    shm_size: "128mb"
+    working_dir: /opt/yucca
+    volumes:
+      - "${APP_DATA_DIR}/data/data:/opt/yucca/data"
+      - "${APP_DATA_DIR}/data/ffmpeg:/opt/yucca/ffmpeg"
+    ports:
+      - 10925:10925 # SMTP server
```

Comment:
I see on your [official documentation](https://docs.yucca.app/en/install/Docker/) that you also need to expose ports for Pprof and Telemetry here. Both 9911 and 9912 are available on the host.

Reply:
Pprof for debug, but Telemetry i added

Reply:
Thanks. I checked and port 9912 is free 👌